src/index.ts

```diff
@@ -130,21 +130,88 @@ export default function register(vjs: typeof videojs = videojs) {
 
             try {
                 display = this.getDisplayMedia(response);
+                let adParameters = '';
+                let hasNestedVast = false;
+
+                do {
+                    hasNestedVast = false;
+                    adParameters = display.creative.adParameters || '';
+
+                    // This is a verizon o2shim "https://acds.prod.vidible.tv/o2shim"
+                    if (display.media?.fileURL?.includes('acds.prod.vidible.tv/o2shim')) {
```

Comment:
An improvement could be to parse out the file url into a url object, then switch on the hostname